samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -69,11 +69,24 @@ static const struct bt_data ad[] = {
 	BT_DATA(BT_DATA_SVC_DATA16, unicast_server_addata, ARRAY_SIZE(unicast_server_addata)),
 };
 
+#define AUDIO_DATA_TIMEOUT_US 1000000UL /* Send data every 1 second */
+#define SDU_INTERVAL_US       10000UL   /* 10 ms SDU interval */
+
 static uint16_t get_and_incr_seq_num(const struct bt_bap_stream *stream)
 {
+	uint16_t seq_num;
```

Comment:
See comment below, but in any case this can be move to the inner scope where it is used and returned

Reply:
Done